com.github.xournalpp.xournalpp.yaml

```diff
@@ -57,10 +57,14 @@ modules:
       - type: archive
         url: https://libzip.org/download/libzip-1.5.2.tar.xz
         sha256: b3de4d4bd49a01e0cab3507fc163f88e1651695b6b9cb25ad174dbe319d4a3b4
+
   - name: libportaudiocpp
     buildsystem: autotools
     config-opts:
       - "--enable-cxx" # compile with c++ headers!!!
```

Comment:
This comment is also not really descriptive. Something like "Xournal++ uses the C++ headers of PortAudio, so compile them" would be better.